backend/mobileserver/implicit.go

```diff
@@ -0,0 +1,9 @@
+// This file imports implicit dependencies required by generated code.
```

Comment:
Maybe clearer wording: "This file imports the packages required by gomobile so they can be vendored".

Reply:
Done.